frontend/src/styles/base/global.scss

```
@@ -44,3 +49,80 @@ img {
   display: none !important;
   visibility: hidden !important;
 }
+
+.hidden {
+  &-sm {
+    @include sm-block {
+      display: none;
+    }
+  }
+  &-md {
+    @include md-block {
+      display: none;
+    }
+  }
+}
+
+.text {
+  &-left {
+    text-align: left;
+  }
+  &-center {
+    text-align: center;
+  }
+  &-right {
+    text-align: right;
+  }
+}
```

Example
```suggestion
.Text {}
.Text.Text--left {}
.Text.Text--center {}
.Text.Text--right {}
.Text.Text--uppercase {}
```

И дальше можешь комбинировать в HTML как
<p class="Text Text--center Text--uppercase">

Читабельнее, по BEM, искабельно :)